src/android/com/gae/scaffolder/plugin/FCMPlugin.java

```diff
@@ -271,7 +271,10 @@ public void run() {
                 try {
                     if (Build.VERSION.SDK_INT >= 33) { // Android 13+
```

Comment:
What if SKD is lower than 33? There should be else block with `callbackContext.success(1);`.

Reply:
You are right it seems ok to me, would you like to apply the fix?

Reply:
The fix is applied in [this PR](https://github.com/marutifh/cordova-plugin-fcm-with-dependecy-updated/pull/4/files)